Changed nav-bar to Offcanvas nav-bar #31

rajuljha opened the pull request:
Created new feature (#31). Changed the navBar to Offcanvas navBar for both computers and mobiles. 
Removed the previous navBar code and added new Offcanvas navbar template.

pythoniclaver commented:
> Created new feature (#31). Changed the navBar to Offcanvas navBar for both computers and mobiles. Removed the previous navBar code and added new Offcanvas navbar template.

@rajuljha Thanks for wanting to add new feature to our application but the our rules clearly said we want to keep this application simple, No Framework, No packages, No Libraries.

I would have even merged it if your PR was bug free but it isn't.

Please stay updated as I will be creating other issues that you might be able to fix, I will close this PR now.

rajuljha commented:
> > Created new feature (#31). Changed the navBar to Offcanvas navBar for both computers and mobiles. Removed the previous navBar code and added new Offcanvas navbar template.
> 
> @rajuljha Thanks for wanting to add new feature to our application but the our rules clearly said we want to keep this application simple, No Framework, No packages, No Libraries.
> 
> I would have even merged it if your PR was bug free but it isn't.
> 
> Please stay updated as I will be creating other issues that you might be able to fix, I will close this PR now.

@pythoniclaver I understand your concerns and I appreciate you for letting me contribute to the project by assigning this issue to me. I learnt a lot through this project. Hoping I can contribute to the project in the future. Thank You!
